src/layouts/DialogLayout/DialogLayout.stories.tsx

```diff
@@ -55,12 +55,10 @@ export const DialogOnboarding: Story = {
                 activeKey="1"
                 onSwitch={() => {}}
               />
-              {/* TODO: Maybe this should be a separate component? */}
-              <hr/>
-              <div style={{display: 'flex', flexDirection: 'row', gap: '4px', alignItems: 'flex-start'}}>
-                <p><Icon icon="info"/></p>
-                <p><small>Step-by-Step Guide to Connect to Azure Cloud Provider</small></p>
-              </div>
+              <p className={cl['bk-dialog-layout__content__aside__hint']}>
```

Comment:
Instead of using `DialogLayoutClassNames` (which is not exported), we can add a `DialogLayout.Hint` subcomponent.